atv/settings.py

```diff
@@ -171,10 +170,7 @@
 CORS_ORIGIN_ALLOW_ALL = env.bool("CORS_ORIGIN_ALLOW_ALL")
 
 REST_FRAMEWORK = {
-    "DEFAULT_AUTHENTICATION_CLASSES": [
-        "helusers.oidc.ApiTokenAuthentication",
-        "rest_framework.authentication.SessionAuthentication",
-    ],
+    "DEFAULT_AUTHENTICATION_CLASSES": ["helusers.oidc.ApiTokenAuthentication"],
```

Comment:
Can we keep the session authentication at least for testing? Since it's easier when logging in as a Django superuser